routers/frr.php

```diff
@@ -19,11 +19,31 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
  */
 
-require_once('quagga.php');
+require_once('unix.php');
 require_once('includes/command_builder.php');
 require_once('includes/utils.php');
 
-final class FRR extends Quagga {
+class FRR extends UNIX {
+  protected static $wrapper = 'vtysh -c';
+
+  protected function build_bgp($parameter) {
+    $cmd = new CommandBuilder();
+    // vytsh commands need to be quoted
+    $cmd->add(self::$wrapper, '"', 'show');
+	
+	$cmd->add('bgp');
```

Comment:
You can remove this and add the `bgp` word inside the previous call to `add()` like `$cmd->add(self::$wrapper, '"', 'show bgp');`.